src/reducers/org.js

```diff
@@ -972,7 +972,7 @@ const addNewPlanningItem = (state, action) => {
   const newPlanningItem = fromJS({
     id: generateId(),
     type: action.planningType,
-    timestamp: timestampForDate(new Date()), // TODO: impure new Date()
+    timestamp: timestampForDate(action.timestamp),
```

Comment:
Is there a naming convention for date/time/timestamp, ...? `timestamp` in Organice is mostly used for the parse structure of a timestamp (see timestamp.js, `getCurrentTimestamp` and so on). So I'd suggest `currentDate` for a `new Date()` timestamp. Or is there another convention in JS on how to call these? Like `dateTime` or `time`?

Reply:
I agree that `currentDate` would be a better name overall for this.

The reason I named it `timestamp` is because we're assining it to another attribute that already has the name `timestamp`. So I was working on removing the `TODO` regarding impurity in the reducer and didn't want to go down the rabbit hole of finding all the `timestamp`s that should be `currentDate`. That's better done in a second step. But I agree, it would be much more appropriate naming. In general in JS, but specifically in organice since a `timestamp` already is a well known entity.